### PR TITLE
refactor: GreptimeDB Operator section

### PR DESCRIPTION
**Pull request by WenyXu**
## What's Changed in this PR

Refactor GreptimeDB Operator section

- [x] All new English content is ready
- [x] Remove `/user-guide/operations/deploy-on-kubernetes/greptimedb-operator.md`
- [x] Add zh documents
- [x] Remove `greptimedb-operator.md` from the zh documentation
- [x] Fix dead links, if exist

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `sidebars.ts` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Upgrade GreptimeDB Operator
+You can upgrade the GreptimeDB Operator at any time without impacting your managed GreptimeDB instances. This guide details the steps to upgrade an existing GreptimeDB Operator installation using Helm, specifically from version 0.2.1 to 0.2.3.
+
+### 1. Verify the existing Operator installation.
```
Comment (nicecui) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md:4]:
```suggestion
### Verify the existing Operator installation
```

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+```bash
+helm list -n greptimedb-operator
+```
+
+You should see output similar to this:
+```
+NAME               	NAMESPACE          	REVISION	UPDATED                                	STATUS  	CHART                    	APP VERSION
+greptimedb-operator	greptimedb-operator	1       	2024-08-30 08:04:53.388756424 +0000 UTC	deployed	greptimedb-operator-0.2.1	0.1.0-alpha.28
+```
+
```
Comment (nicecui) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md:25]: please explain the version from the output

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+### 2. Update the Operator Repository
```
Comment (nicecui) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md:26]:
```suggestion
### Update the Operator repository
```

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+```bash
+helm search repo greptimedb-operator
+```
+
+You should see output similar to this:
+```bash
+NAME                        	CHART VERSION	APP VERSION   	DESCRIPTION
+greptime/greptimedb-operator	0.2.3        	0.1.0-alpha.29	The greptimedb-operator Helm chart for Kubernetes.
+```
+
```
Comment (nicecui) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md:51]: please explain the version from the output

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
+# Upgrade GreptimeDB Operator
+You can upgrade the GreptimeDB Operator at any time without impacting your managed GreptimeDB instances. This guide details the steps to upgrade an existing GreptimeDB Operator installation using Helm, specifically from version 0.2.1 to 0.2.3.
```
Comment (nicecui) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md:2]:
```suggestion
You can upgrade the GreptimeDB Operator at any time without impacting your managed GreptimeDB instances. This guide details the steps to upgrade an existing GreptimeDB Operator installation using Helm, specifically from version 0.2.1 to the lastest version.
```

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+helm search repo greptimedb-operator
+```
+
+You should see output similar to this:
+```bash
+NAME                        	CHART VERSION	APP VERSION   	DESCRIPTION
+greptime/greptimedb-operator	0.2.3        	0.1.0-alpha.29	The greptimedb-operator Helm chart for Kubernetes.
+```
+
+### 3. Run helm upgrade
```
Comment (nicecui) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md:52]:
```suggestion
### Upgrade the Operator version
```

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+greptime/greptimedb-operator	0.2.3        	0.1.0-alpha.29	The greptimedb-operator Helm chart for Kubernetes.
+```
+
+### 2. Install the Operator
+To install the Operator, run the following `helm install` command. This command also creates a dedicated namespace `greptimedb-operator` for the installation. It's recommended to use a dedicated namespace for the Operator:
+```bash
+helm install \
+  --create-namespace \
+  greptimedb-operator greptime/greptimedb-operator \
+  -n greptimedb-operator
```
Comment (nicecui) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md:39]:
```suggestion
helm install \
  greptimedb-operator greptime/greptimedb-operator \
  --create-namespace \
  -n greptimedb-operator
```

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+Use Helm to upgrade the GreptimeDB Operator to the latest version:
+```bash
+helm upgrade -n greptimedb-operator \
+operator greptime/greptimedb-operator
```
Comment (nicecui) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md:57]:
```suggestion
helm upgrade -n greptimedb-operator \
greptimedb-operator greptime/greptimedb-operator
```

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+```
+
+If the Operator is installed in a different namespace, specify it with the `-n` argument. Additionally, if you used a different installation name than operator, replace `operator` in the command.
```
Comment (nicecui) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md:60]:
```suggestion
If the Operator is installed in a different namespace, specify it with the `-n` argument. Additionally, if you used a different installation name than operator, replace `greptimedb-operator` in the command.
```

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+
+The command should return a successful upgrade with an incremented `REVISION` value.
+
+### 4. Validate the Operator upgrade
```
Comment (nicecui) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md:64]:
```suggestion
### Verify the Operator upgrade
```

**Review by nicecui — COMMENTED** [review]
There is some new content about CRDs in https://docs.greptime.com/nightly/user-guide/operations/deploy-on-kubernetes/greptimedb-operator, please also add those content to this PR as well

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+## Overview
+The GreptimeDB Kubernetes Operator simplifies deploying GreptimeDB on both private and public cloud infrastructures. This guide walks you through installing the latest stable version of the GreptimeDB Operator on a Kubernetes cluster. The Operator leverages a [Custom Resource Definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) to represent GreptimeDB as a Kubernetes [object](https://kubernetes.io/docs/concepts/overview/working-with-objects/).
+
+
+## Prerequisites
+- [Helm](https://helm.sh/docs/intro/install/) (Use the Version appropriate for your Kubernetes API version)
+
+## Install the GreptimeDB Operator using Helm Charts
+
+GreptimeDB provides a Helm-compatible repository for easy deployment. Follow these steps to install the Operator using Helm:
```
Comment (daviderli614) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md:12]: suggestion: add greptime helm-chart repository link ?

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+greptime/greptimedb-operator	0.2.3        	0.1.0-alpha.29	The greptimedb-operator Helm chart for Kubernetes.
+```
+
+### Install the Operator
+To install the Operator, run the following `helm install` command. This command also creates a dedicated namespace `greptimedb-operator` for the installation. It's recommended to use a dedicated namespace for the Operator:
+```bash
+helm install \
+  operator greptime/greptimedb-operator \
+  --create-namespace \
+  -n greptimedb-operator
```
Comment (daviderli614) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md:37]: suggestion:
```
helm upgrade \
  --install \
  --create-namespace \
  greptimedb-operator greptime/greptimedb-operator \
  -n greptimedb-admin
  ```
  same as: https://github.com/GreptimeTeam/helm-charts

Comment (WenyXu) [reply docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md:37]: I prefer to keep the `install` command and the `greptimedb-operator` here.

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+```
+
+### Verify the Operator installation
+After installation, check the contents of the `greptimedb-operator` namespace to confirm that all pods and services are running correctly:
+```bash
+kubectl get all -n greptimedb-operator
+```
+
+You should see output similar to this:
```
Comment (daviderli614) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md:46]: You cam see output similar to this:

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md [hunk 14]
```diff
@@ -0,0 +1,76 @@
...
+NAME                        	CHART VERSION	APP VERSION   	DESCRIPTION
+greptime/greptimedb-operator	0.2.3        	0.1.0-alpha.29	The greptimedb-operator Helm chart for Kubernetes.
+```
+
+### Upgrade the Operator version
+
+Use Helm to upgrade the GreptimeDB Operator to the latest version:
+```bash
+helm upgrade -n greptimedb-operator \
+operator greptime/greptimedb-operator
```
Comment (nicecui) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md:59]:
```suggestion
helm upgrade -n greptimedb-operator \
  operator greptime/greptimedb-operator
```

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md [hunk 15]
```diff
@@ -0,0 +1,56 @@
...
+greptime/greptimedb-operator	0.2.3        	0.1.0-alpha.29	The greptimedb-operator Helm chart for Kubernetes.
+```
+
+### Install the Operator
+To install the Operator, run the following `helm install` command. This command also creates a dedicated namespace `greptimedb-operator` for the installation. It's recommended to use a dedicated namespace for the Operator:
+```bash
+helm install \
+  operator greptime/greptimedb-operator \
+  --create-namespace \
+  -n greptimedb-operator
```
Comment (zyy17) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md:37]: maybe use `greptimedb-admin` will be better. Keep constitent with other docs.

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+```
+
+### Verify the Operator installation
```
Comment (zyy17) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md:40]: Can we add the process of validating the CRD installation?

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md [hunk 17]
```diff
@@ -0,0 +1,56 @@
...
+After installation, check the contents of the `greptimedb-operator` namespace to confirm that all pods and services are running correctly:
+```bash
+kubectl get all -n greptimedb-operator
```
Comment (zyy17) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md:43]: To simplify the output, it's better to use `get pods`. The users only need to know the status of the operator pod.

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md [hunk 18]
```diff
@@ -0,0 +1,76 @@
+# Upgrade GreptimeDB Operator
+You can upgrade the GreptimeDB Operator at any time without impacting your managed GreptimeDB instances. This guide details the steps to upgrade an existing GreptimeDB Operator installation using Helm, specifically from version 0.2.1 to the latest version.
+
+### Verify the existing Operator installation.
+
+First, verify the health and status of all Operator pods and services using:
+
+```bash
+kubectl get all -n greptimedb-operator
```
Comment (zyy17) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md:9]: ditto, `get pods` will be better.

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md [hunk 19]
```diff
@@ -0,0 +1,56 @@
...
+```
+NAME                        	CHART VERSION	APP VERSION   	DESCRIPTION
+greptime/greptimedb-operator	0.2.3        	0.1.0-alpha.29	The greptimedb-operator Helm chart for Kubernetes.
+```
+
+### Install the Operator
+To install the Operator, run the following `helm install` command. This command also creates a dedicated namespace `greptimedb-operator` for the installation. It's recommended to use a dedicated namespace for the Operator:
+```bash
+helm install \
+  operator greptime/greptimedb-operator \
```
Comment (zyy17) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/deploy-greptimedb-operator.md:35]: It's better to use `greptimedb-operator` as release nanem.

File: docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md [hunk 20]
```diff
@@ -0,0 +1,76 @@
...
+```
+
+If the Operator was installed in a custom namespace, replace `greptimedb-operator` with your specific namespace using `-n <NAMESPACE>`.
+
+Next, view the installed Helm charts in the namespace:
+
+```bash
+helm list -n greptimedb-operator
```
Comment (zyy17) [inline docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md:17]: Ditto, you can use:

```
helm list greptimedb-operator -n greptimedb-admin
```

Comment (WenyXu) [reply docs/user-guide/operations/deploy-on-kubernetes/manage-greptimedb-operator/upgrade-greptimedb-operator.md]: Resolved in [9459ef8](https://github.com/GreptimeTeam/docs/pull/1160/commits/9459ef835a33c298279c5249dacff936f87d056f)

**Review by nicecui — COMMENTED** [review]
please remove `greptimedb-operator.md` and fix dead links

**Review by nicecui — APPROVED** [review]

**Review by daviderli614 — APPROVED** [review]

**Pull request merged**